hca/define_api.py

```diff
@@ -1,37 +1,165 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-import json, sys, os
+import json, sys, os, ast
```

Comment:
Will do. Should I not do one import per line if coming from same module? Also seen standards where w/in package imports are below other imports. Is this something we do as well?